### DISCUSSION TITLE
Directory-Breadcrumb mapping

### DISCUSSION THREAD

**Discussion by thdoan** [lumeland/lume#627] [Q&A] [answered]
Would it be possible to have a directory called `about` that is mapped to "About Us" when the breadcrumb is rendered? I couldn't figure out a way to do this. For example, the Simple Wiki theme uses `item.slug` for the directory when rendering the breadcrumb, so you end up with an unaesthetic breadcrumb:

My Site / about /

What I would like to do is have a breakcrumb that looks like this:

My Site / About Us /

and the "About Us" page links to /about/ (not /about-us/).

**Comment by oscarotero** [comment, marked as answer]
The theme uses `item.slug` if there isn't any `index.html` file in the directory. Otherwise it uses the page title.
If your url structure is, for example:

```
/
/about/
/about/team/
/about/team/laura/
```

The breadcrumb should display the title page of every directory. But if there are intermediate directories, without pages, like

```
/
/about/
/about/team/laura/
```

The folder `/about/team/` doesn't have a page, so the breadcrumb doesn't know what to show and use the slug. See [the documentation for more info](https://lume.land/plugins/nav/#menu).

**Comment by thdoan** [comment]
@oscarotero what you explained makes sense -- in my initial testing I didn't create an index page for `/about/`.
